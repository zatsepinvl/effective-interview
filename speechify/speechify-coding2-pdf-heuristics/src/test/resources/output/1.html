            <html><body>
            <div style="display: flex; flex-direction: row">
            <div style="width: 50%">
            <h1>Expected</h1>
            <hr />
            <h2>AUSTRALIAN EJOURNAL OF THEOLOGY Issue 4 - February 2005</h2>
<h2>RENEWING CHRISTIAN ANTHROPOLOGY IN TERMS OF KASHMIR SHAIVISM</h2>
<h2>JOHN DUPUCHE</h2>
<h2>Introduction</h2>
<h2>AUSTRALIAN EJOURNAL OF THEOLOGY Issue 4 - February 2005 RENEWING CHRISTIAN ANTHROPOLOGY IN TERMS OF KASHMIR SHAIVISM JOHN DUPUCHE Introduction</h2>
<p>My good friend Bettina Bäumer1 relates the following story:</p>
<p>“It was a seminar in Vienna University where [Karl Rahner] also spoke and I gave my first ever paper on KS [Kashmir Shaivism] on anupÁya. After listening very attentively, he took me aside after the discussion and said [Wir sind nur Waisenkinder] [which she glosses as] “we are orphans compared to what these Indians have discovered!” (Waisenkinder means we are far behind or more primitive, spiritually).”2</p>
<p>The first generations of Christians moved out of the Jewish framework into the thought-world of the Greeks and reinterpreted their faith in a new way. Now with the end of the colonial era, where the East was interesting only if it was exotic, we are witnessing a massive new shift. Rahner’s comment to Bettina Bäumer reflects his awareness that the Hindu thought must profoundly affect Christian theology, making Christians qualify categories and images that are so familiar as to be unquestioned.</p>
<p><i>1 Prof. Dr. Bettina Bäumer, Institute of Religious Studies, University of Vienna. 2 Personal communication, 9 April 2004.</i></p>
<p><i>1</i></p>
            </div>
            <div style="width: 50%">
            <h1>Actual</h1>
            <hr />
            
            </div>
            </div>
            </body></html>