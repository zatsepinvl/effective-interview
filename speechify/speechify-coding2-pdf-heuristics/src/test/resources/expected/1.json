[
  {
    "type": "Heading",
    "text": "AUSTRALIAN EJOURNAL OF THEOLOGY"
  },
  {
    "type": "Heading",
    "text": "Issue 4 - February 2005"
  },
  {
    "type": "Heading",
    "text": "RENEWING CHRISTIAN ANTHROPOLOGY IN TERMS OF KASHMIR SHAIVISM"
  },
  { "type": "Heading", "text": "JOHN DUPUCHE" },
  { "type": "Heading", "text": "Introduction" },
  {
    "type": "Paragraph",
    "text": "My good friend Bettina B\u00e4umer1 relates the following story:"
  },
  {
    "type": "Paragraph",
    "text": "\u201cIt was a seminar in Vienna University where [Karl Rahner] also spoke and I gave my first ever paper on KS [Kashmir Shaivism] on anup\u00c1ya. After listening very attentively, he took me aside after the discussion and said [Wir sind nur Waisenkinder] [which she glosses as] \u201cwe are orphans compared to what these Indians have discovered!\u201d (Waisenkinder means we are far behind or more primitive, spiritually).\u201d2"
  },
  {
    "type": "Paragraph",
    "text": "The first generations of Christians moved out of the Jewish framework into the thought-world of the Greeks and reinterpreted their faith in a new way. Now with the end of the colonial era, where the East was interesting only if it was exotic, we are witnessing a massive new shift. Rahner\u2019s comment to Bettina B\u00e4umer reflects his awareness that the Hindu thought must profoundly affect Christian theology, making Christians qualify categories and images that are so familiar as to be unquestioned."
  },
  {
    "type": "Marginalia",
    "text": "1 Prof. Dr. Bettina B\u00e4umer, Institute of Religious Studies, University of Vienna. 2 Personal communication, 9 April 2004."
  },
  { "type": "Marginalia", "text": "1" }
]
